release-notes/release-5.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: April 17, 2024
+---
+
+# Features
+
+- [Oracle Database Integration]() with default dashboard
+- [Apache Networking Integration]() with default dashboard
+- Automated log collection with the Vercel APM for __Build__, __Lambda__, __Static__, __Edge__, and __External__ immediately upon installation
+- Ability to parse attributes from non-standard JSON log messages
+- Improved sharing of RUM links with filters as query parameters within RUM to easily share user-specific sessions with your teammates
```

Comment:
Can remove everything from "...within RUM..." onwards, because the notion of sharing links implies sharing with others